src/test/scala/com/rbmhtechnology/calliope/EventDeletionSpec.scala

```diff
@@ -104,25 +111,24 @@ class EventDeletionSpec extends TestKit(ActorSystem("test"))
             pub.expectRequest()
           }
           "invoke deleteEvents for each time frame" in {
-            val (pub, _) = runSink(EventDeletion.sink[SequenceNr](actorEventDeleter(), perDuration = 500.millis))
+            val deleteInterval = 500.millis
```

Comment:
I wonder if we could reduce this a little bit to reduce test-time. Is 100 feasible?